src/NotificationBar/NotificationBar.styles.ts

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components';
+import { NotificationIcon, UpgradeIcon } from '../Icons';
+import { Props } from './NotificationBar';
+
+export const BarDiv = styled.div<Props>`
+  width: 100%;
+  min-height: 40px;
+  padding: 5px;
+  border: 1px solid;
+  display: flex;
+  justify-content: space-between;
+  ${p =>
+    p.variant === 'refresh' &&
```

Comment:
I think we should set the `color` in here too, that way it will default to the correct (if a string is passed in for example) but can still be overwritten by passing in a component with styles. Thoughts?

Reply:
I could do with a call on this one to make it clear in my head